packages/malloy-vscode/src/extension/webviews/query_page/App.tsx

```diff
@@ -98,6 +100,11 @@ export const App: React.FC = () => {
               setStatus(Status.Displaying);
               setTimeout(() => {
                 setHTML(rendered);
+                if (data.rowCount < result.totalRows) {
+                  setWarning(
+                    `Row limit hit. Viewing ${data.rowCount} of ${result.totalRows} results.`
```

Comment:
Might be nice to use `.toLocaleString()` for these to get the nice comma thousands separators.